examples/001-dotnet-WebClient/README.md

```diff
@@ -3,12 +3,19 @@
 This example shows how to import multiple files and ask questions, delegating
 the work to the Kernel Memory Service.
 
-The example points to `https://127.0.0.1:9001` so you should start the
-Kernel Memory Service locally before running this example.
+### There are two ways to run this example:
+
+1. The example points to `https://127.0.0.1:9001` so you should start the
+   Kernel Memory Service locally before running this example.
+
+2. You can easily deploy Kernel Memory to Azure by following procedure on [this page](../../infra/README.md).
```

Comment:
I'd move this down or revisit the README, because one needs also to change the URL when working with the service running on Azure. One way would be splitting the doc in two parts "running the example locally" and "running the example with KM on Azure"